bundles/org.openhab.binding.robonect/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -45,6 +45,10 @@
 
 			<channel id="health-temperature" typeId="temperatureType"/>
 			<channel id="health-humidity" typeId="humidityType"/>
+
+			<channel id="blades-quality" typeId="bladesQualityType"/>
+			<channel id="blades-days" typeId="bladesQualityDaysType"/>
```

Comment:
Have you considered whether to express this as number of days since last replacement, or simply as last replacement date? I'm not saying one is "better" than the other, but it might be worth at least considering. Number of days will change each day while a replacement day (which of course would be calculated) would be fixed until next replacement. I guess it also depends on the typical usage which would be preferable.

Reply:
Might be a better solution, but i don't want to change this too much as i;m not the original author. Think it is mergable now.

Reply:
It is not something that i have chosen - that is how the hardware works and what the API reports.

The hardware also allows to set the maximum usage hours and maximum age for the blades - when exceeded will show a popup on the web UI (but that's not exposed over the API afaik)